datadog_checks_dev/datadog_checks/dev/tooling/e2e/local.py

```diff
@@ -79,6 +79,9 @@ def agent_command(self):
         return get_agent_exe(self.agent_version, platform=self.platform)
 
     def exec_command(self, command, **kwargs):
+        if command.startswith('pip '):
+            command = command.replace('pip ', get_pip_exe(self.python_version, self.platform), 1)
```

Comment:
We should fix get_pip_exe for MAC, otherwise it won't work for the majority of us.

Reply:
This exec_command is to run commands in docker not on the local machine, right?

Reply:
Well no this is local environment support.

Reply:
Oh, sorry, there is another exec_command in docker.py

Reply:
Yeah, for local environment should it use the tox pip rather than the agent pip?